src/App.tsx

```diff
@@ -20,7 +20,7 @@ import {
   web3Enable,
   web3FromAddress
 } from '@polkadot/extension-dapp';
-import keyring from '@polkadot/ui-keyring';
+// import keyring from '@polkadot/ui-keyring';
```

Comment:
Very minor, but I'd prefer to remove this rather than comment it out.

Reply:
I do agree.

Reply:
Agree! Missed this when committing. Thanks!